docs/source/tutorial/queries.mdx

```diff
@@ -121,132 +209,162 @@ const Launches: React.FC<LaunchesProps> = () => {
     GetLaunchListTypes.GetLaunchList,
     GetLaunchListTypes.GetLaunchListVariables
   >(GET_LAUNCHES);
-  // same as above
+  const [isLoadingMore, setIsLoadingMore] = useState(false); //highlight-line
```

Comment:
You should be able to make `fetchMore` produce an initial loading state by configuring `useQuery(GET_LAUNCHES, { notifyOnNetworkStatusChange: true })`, so you don't need this React `useState` logic.

Reply:
ok cool! I'll circle back to this with a separate change after this big change is out, just to make sure I don't flub the accompanying changes to the sample repo